api/src/routes/openai.ts

```diff
@@ -22,6 +22,8 @@ const FRIENDLY_PARAMETER_GENERATION_SYSTEM_PROMPT = cleanPrompt(`
   Use the tool "make_request". Always respond in valid JSON.
 `);
 
+// NOTE - I had to stop instructing the AI to create very long data.
```

Comment:
had to remove the instruction to test "exteremely long data" as this would break function calling by exceeding our token limit in our response